src/app/reset-password/page.tsx

```diff
@@ -0,0 +1,139 @@
+'use client';
+import React, { useState } from 'react';
+
+const ResetPassword = () => {
+  const [toggleEye, setToggleEye] = useState(false);
+
+  const toggleEyeHandler = () => {
+    setToggleEye((prev) => !prev);
+  };
+
+  return (
+    <>
+      <div className="my-7 md:mt-2  md:mb-14 h-[calc(100vh-16.5rem)] md:h-[calc(100vh-19rem)] bg-[#EEE7DF] md:bg-white">
+        <div className="w-[90%] md:w-[80%] md:px-10 lg:w-[60%] mx-auto md:bg-[#EEE7DF]">
```

Comment:
for customized widths and heights like w-[90%], use the nearest ones or if you need the exact percentage, in the tailwind.config underneath colors you can add width: {'80p': '80%'}. p is the naming convention we'll use to identify it's percentage when using it. same thing for height